bin/rebuild-for-deploy.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -ex
+script_dir=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
+cd $script_dir || exit 1
+cd .. || exit 1
+
+rm -rf node_modules
+rm -f package-lock.json
+npm install -omit=dev
```

Comment:
we're reinstalling node_modules anyway? even though its checked in?

Reply:
This is a helper file for developers.  There is no CI/CD.
I took a light touch to this update.
The README make a half-hearted attempt to explain this.